docs/system-requirements.md

```diff
@@ -45,8 +45,25 @@ The specific storage consumption for k0s is as follows:
 
 ## Host operating system
 
-- Linux (see [Linux specific requirements] for details)
-- Windows Server 2019
+k0s is tested and known to work on the following operating systems and versions:
```

Comment:
Why not keep the general claim that it works on Linux and Windows, and then list the automatically tested distributions below? If you only mention the test matrix, people might get the impression that k0s only supports the listed distributions, don't you think?